### PR TITLE
Fix mutect2 pileup wrapper. Fix log files for mutect2 steps.

### PR DESCRIPTION
**Pull request by messersc**
Hi @ericblanc20,

After you have done your changes on somatic-update branch, you can merge this on top of it and then we can merge the changes of both into main. Let me know if you need another review by then.

Cheers,
Clemens

### CODE REVIEW

**Comment by eudesbarbosa** [comment]
@messersc, please fix the CI before merging: 
`isort --force-sort-within-sections -profile`

**Comment by messersc** [comment]
Hi @eudesbarbosa,

thanks for the hint. We will merge this onto @ericblanc20 s `somatic-update` branch first and then we will make the linter and tests happy before we merge into main.

Clemens

File: snappy_wrappers/wrappers/mutect2/contamination/wrapper.py [hunk 1]
```diff
@@ -18,13 +18,13 @@
...
     if [[ "$(set +e; tty; set -e)" != "" ]]; then
-        rm -f "{snakemake.log}" && mkdir -p $(dirname {snakemake.log})
-        exec &> >(tee -a "{snakemake.log}" >&2)
+        rm -f "{snakemake.log.log}" && mkdir -p $(dirname {snakemake.log.log})
+        exec &> >(tee -a "{snakemake.log.log}" >&2)
     else
-        rm -f "{snakemake.log}" && mkdir -p $(dirname {snakemake.log})
-        echo "No tty, logging disabled" >"{snakemake.log}"
+        rm -f "{snakemake.log.log}" && mkdir -p $(dirname {snakemake.log.log})
+        echo "No tty,.log.logging disabled" >"{snakemake.log.log}"
```
Comment (eudesbarbosa) [inline snappy_wrappers/wrappers/mutect2/contamination/wrapper.py:27]: Does this message make more sense than the previous one?

File: snappy_wrappers/wrappers/mutect2/filter/wrapper.py [hunk 2]
```diff
@@ -18,13 +18,13 @@
...
     if [[ "$(set +e; tty; set -e)" != "" ]]; then
-        rm -f "{snakemake.log}" && mkdir -p $(dirname {snakemake.log})
-        exec &> >(tee -a "{snakemake.log}" >&2)
+        rm -f "{snakemake.log.log}" && mkdir -p $(dirname {snakemake.log.log})
+        exec &> >(tee -a "{snakemake.log.log}" >&2)
     else
-        rm -f "{snakemake.log}" && mkdir -p $(dirname {snakemake.log})
-        echo "No tty, logging disabled" >"{snakemake.log}"
+        rm -f "{snakemake.log.log}" && mkdir -p $(dirname {snakemake.log.log})
+        echo "No tty,.log.logging disabled" >"{snakemake.log.log}"
```
Comment (eudesbarbosa) [inline snappy_wrappers/wrappers/mutect2/filter/wrapper.py:27]: idem

File: snappy_wrappers/wrappers/mutect2/pileup/wrapper.py [hunk 3]
```diff
@@ -21,13 +26,13 @@
...
     if [[ "$(set +e; tty; set -e)" != "" ]]; then
-        rm -f "{snakemake.log}" && mkdir -p $(dirname {snakemake.log})
-        exec &> >(tee -a "{snakemake.log}" >&2)
+        rm -f "{snakemake.log.log}" && mkdir -p $(dirname {snakemake.log.log})
+        exec &> >(tee -a "{snakemake.log.log}" >&2)
     else
-        rm -f "{snakemake.log}" && mkdir -p $(dirname {snakemake.log})
-        echo "No tty, logging disabled" >"{snakemake.log}"
+        rm -f "{snakemake.log.log}" && mkdir -p $(dirname {snakemake.log.log})
+        echo "No tty,.log.logging disabled" >"{snakemake.log.log}"
```
Comment (eudesbarbosa) [inline snappy_wrappers/wrappers/mutect2/pileup/wrapper.py:35]: idem

**Comment by eudesbarbosa** [comment]
Probably not a good practice. I'd argue that is better to fix errors before pushing to the larger more complex branch. Also, the simple style mistake breaking CI could be masking more relevant errors.

**Comment by messersc** [comment]
Understood, I will fix it. 
Thanks Eudes!

**Pull request merged**